<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_emoji</class>
 <widget class="QDialog" name="Dialog_emoji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>229</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Emoji</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
